node/_tools/config.json

```diff
@@ -35,6 +35,7 @@
       "test-stream-readable-event.js",
       "test-stream-writable-change-default-encoding.js",
       "test-url-format-whatwg.js",
+      "test-url-parse-format.js",
```

Comment:
ref: https://github.com/denoland/deno_std/pull/1667/files#diff-2dc606fcc5aca07ba82c64a68167b04a1af3fbada738bd90d8cd499d81de580a

Reply:
This is correct. Thanks!